### ISSUE TITLE
New contributors

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6614]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#6614|
|Type|enhancement|
|Created|2013-07-05 00:50:03Z|
|Branch|https://github.com/twisted/twisted/tree/contributors-6614|

The following regular contributors are missing from the [LICENSE](LICENSE) file: Julian, tom.prince, rwall, and ashfall (sorry if I missed anyone). I'm not sure what the policy is regarding credits in that file so.. ill make it an enhancement request.

<details><summary>Searchable metadata</summary>

```
trac-id__6614 6614
type__enhancement enhancement
reporter__thijs thijs
priority__highest highest
milestone__ 
branch__branches_contributors_6614 branches/contributors-6614
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1372985403000000 1372985403000000
changetime__1390008313000000 1390008313000000
version__None None
owner__exarkun exarkun
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

Previous update: [#4220](https://github.com/twisted/twisted/issues/4220)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [41295]) Branching to 'contributors-6614'

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

What's _supposed_ to be the process (and by this I think I mean what _I want_ to be the process) is that new committers commit their name to this file in their first non-"trivial" (where "trivial" is left as an exercise for copyright lawyers who disagree about how big a change has to be to be copyrightable) change.

Maybe one day we can actually have a process for inducting people.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph set owner to @exarkun|
|-|-|

Thank you for the ultimate softball review.

Approved, please land.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [41317]) Merge contributors-6614

Author: exarkun
Reviewer: glyph
Fixes: [#6614](https://github.com/twisted/twisted/issues/6614)

Add some people who have contributed recently to the list of contributors.
